meson_options.txt

```diff
@@ -0,0 +1,2 @@
+option('new-session-manager', type : 'boolean', value : true, description : 'Build new-session-manager GUI')
```

Comment:
I wasn't sure if I should call this `new-session-manager` because that's what the executable is called or `new-session-manager-gui` or just `gui` because it's the GUI for the main program.